tests/std/test.lst

```diff
@@ -222,7 +222,6 @@ tests\P0758R1_is_nothrow_convertible
 tests\P0768R1_spaceship_operator
 tests\P0769R2_shift_left_shift_right
 tests\P0811R3_midpoint_lerp
-tests\P0896R4_P1614R2_comparisons

```

Comment:
You are damaging this line. I suspect that you copied a file instead of applying a patch.